tests/test_bug_snapshot.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+import json
+
+from bugbug.bug_snapshot import rollback
+
+
+def test_bug_snapshot(get_fixture_path):
+    mock_bugs_db_path = get_fixture_path("bugs.json")
+
+    with open(mock_bugs_db_path, "r") as mock_bug_db:
+
+        for i, line in enumerate(mock_bug_db):
+            bug = json.loads(line)
+
+            print(bug["id"])
+            print(i)
```

Comment:
Nit: remove the printing

Reply:
As we are in a loop, I think the prints will be helpful to quickly identify which bug is triggering an error. It's also helpful to check that the mock DB is correctly read by running `pytest -s`

Reply:
OK, sounds good!